prime-router/src/main/resources/db/migration/V61__create_item_table.sql

```diff
@@ -0,0 +1,36 @@
+/*
+This SQL creates the tables of the DB. The Flyway tool applies this migration to create the database
+
+Follow this style guide https://about.gitlab.com/handbook/business-ops/data-team/platform/sql-style-guide/
+use VARCHAR(63) for names in organization and schema
+
+Copy a version of this comment into the next migration
+*/
+
+/*
+    An item can be a covid test result or any other message (HL7 v2), row (CSV), bundle (FHIR), et. al.
+*/
+CREATE TABLE item (
+    -- Key
+                            id SERIAL PRIMARY KEY,
+
+    -- Value
```

Comment:
Would it be appropriate/valid/helpful to add `tracking_id` here. `tracking_id` exists in item_lineage table.